libs/config/common/src/lib/app-routes.ts

```diff
@@ -2,9 +2,9 @@ import { environment, envs } from '@biosimulations/shared/environments';
 import { EndpointLoader, LoadedEndpoints } from './endpointLoader';
 
 export class AppRoutes {
-  private simulatorsApp: string;
-  private dispatchApp: string;
-  private platformApp: string;
+  public simulatorsApp: string;
+  public dispatchApp: string;
+  public platformApp: string;
 
```

Comment:
I think that you could add methods for the utilities links (when not locally routed) - then you can restore the Utilities navigation safely.